client/app/scripts/superdesk-archive/module.js

```diff
@@ -130,6 +130,18 @@ define([
         };
     }
 
+    ArchiveService.$inject = ['desks', 'storage'];
+    function ArchiveService(desks, storage) {
+        this.addTaskToArticle = function (item) {
+            if ((!item.task || !item.task.desk) && desks.activeDeskId && desks.userDesks) {
+                var user = storage.getItem('sess:user');
+                var currentDesk = _.find(desks.userDesks._items, {_id: desks.activeDeskId});
```

Comment:
`desks.getCurrentDesk()` will always give some desk based on default desk rules and user will never be able to create items in Personal Workspace.